readthedocs/projects/tasks.py

```diff
@@ -1146,6 +1150,56 @@ def fileify(version_pk, commit):
         )
 
 
+def _update_intersphinx_data(version, path, commit):
+    """
+    Update intersphinx data for this version
+
+    :param version: Version instance
+    :param path: Path to search
+    :param commit: Commit that updated path
+    """
+    object_file = os.path.join(path, 'objects.inv')
+
+    # These classes are from copied from Sphinx tests
```

Comment:
I like to add the permalink to the source code of the external project in this cases, so we can jump directly when something start failing and check how they upgrade it.